<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>824</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <widget class="QWidgetForPCView" name="widget_web1" native="true"/>
        </item>
        <item>
         <widget class="QWidgetForPCView" name="widget_web2" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1">
        <item>
         <spacer name="horizontalSpacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_control" native="true">
          <widget class="QPushButton" name="pushButton_changeweb1">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>80</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>左侧改变</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_changeweb2">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>80</y>
             <width>71</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>右侧改变</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_url">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>10</y>
             <width>461</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_callfunction">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>80</y>
             <width>71</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>function</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_js">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>40</y>
             <width>461</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_threadfun">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>80</y>
             <width>71</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>thread</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_timerStart">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>130</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>timerStart</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_timerStop">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>130</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>timeStop</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWidgetForPCView</class>
   <extends>QWidget</extends>
   <header location="global">QWidgetForPCView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
